docker-compose.yml

```diff
@@ -0,0 +1,106 @@
+version: '3.8'
+
+services:
+  nginx:
+    image: ghcr.io/ai-cfia/nginx:main
+    container_name: fertiscan-nginx
+    restart: always
+    ports:
+      - "80:80"
+    environment:
+      - BACKEND_PORT=5000
+      - FRONTEND_PORT=3000
+    depends_on:
+      - frontend
+      - backend
+    networks:
+      - fertiscan-network
+
+  frontend:
+    image: ghcr.io/ai-cfia/fertiscan-frontend:main
+    container_name: frontend
+    restart: always
+    ports:
+      - "3000:3000"
+    env_file:
+      - .env
+    depends_on:
+      - postgres
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:3000/health"]
```

Comment:
In the frontend, we don't have code for this.

Reply:
I removed it thank you